src/riak_ql_ddl.erl

```diff
@@ -1017,6 +1095,100 @@ is_insert_valid_test_helper(Table_name, Table_def, Insert) ->
         "    myfamily, myseries, time))"
        ).
 
+insert_sql_columns_empty_fields_test() ->
+    {_DDL, Mod} = is_sql_valid_test_helper("mytab", ?LARGE_TABLE_DEF),
+    FInMod = [ {identifier,[F]} || {[F], _I} <- Mod:get_field_positions() ],
+    FieldsIn = [],
+    ValuesIn = [[{varchar, <<"family1">>},
+            {varchar, <<"series1">>},
+            {timestamp, 1000},
+            {varchar, <<"sunny">>},
+            {double, 37.0}]],
+    {FieldsOut, ValuesOut } = insert_sql_columns(Mod, FieldsIn, ValuesIn),
+    ?assertEqual(FInMod, FieldsOut),
+    ?assertEqual(ValuesIn, ValuesOut).
+
+insert_sql_columns_pk_fields_test() ->
```

Comment:
Only specifying primary key fields is the lower-boundary. A query must specify at minimum the primary keys. Other tests that provide partial field specification do not test at this boundary. Generally, testing should test at the boundaries as well as within the boundary.